app/src/main/java/org/simple/clinic/newentry/clearbutton/ClearFieldImageButton.kt

```diff
@@ -60,14 +62,21 @@ class ClearFieldImageButton(context: Context, attrs: AttributeSet) : AppCompatIm
 
     fieldOriginalPaddingEnd = field.paddingEnd
 
-    Observable.merge(clearableFieldTextChanges(), clearableFieldFocusChanges())
+    disposable = Observable.merge(clearableFieldTextChanges(), clearableFieldFocusChanges())
```

Comment:
Shouldn't `.takeUntil(RxView.detaches(this))` take care of disposing the stream?

Reply:
This just makes it explicit.

Reply:
Right, do we still need to `takeUntil` operator then?

Reply:
I don't know if this behavior could influence other parts in which this component is being used.

Reply:
Alright, let's leave this in for now and then and decide later.